src/include/wiredtiger.in

```diff
@@ -1528,11 +1528,9 @@ struct __wt_session {
 	/*!
 	 * Reset the session handle.
 	 *
-	 * This method resets all cursors associated with this session,
-	 * discards cached resources and resets session statistics.
-	 * The session can be re-used immediately after this call returns.
-	 * If a transaction is running on this session, then this call takes
-	 * no action and return an error.
+	 * This method resets the cursors associated with the session, clears session statistics and
+	 * discards cached resources. No session configurations are modified. WT_SESSION::reset will
```

Comment:
I am okay with it as is. I wonder if it should be even more spelled out:
Session configuration settings are not modified. They are not set back to the default value.